kubernetes/apps/web3/monero/guard/resources/controller.py

```
@@ -358,13 +362,19 @@ def do_GET(self):
             self._send(404, "not found\n", "text/plain")
     def _send(self, status, body, content_type):
         data = body.encode()
-        self.send_response(status); self.send_header("Content-Type", content_type); self.send_header("Content-Length", str(len(data))); self.end_headers(); self.wfile.write(data)
+        self.send_response(status)
+        self.send_header("Content-Type", content_type)
+        self.send_header("Content-Length", str(len(data)))
+        self.end_headers()
+        self.wfile.write(data)
     def log_message(self, format, *args):
```

Ruff (0.16.1)

[error] Function argument `format` is shadowing a Python builtin

(A002)